traefik/templates/deployment.yaml

```diff
@@ -23,6 +23,7 @@ spec:
       {{- with .Values.rollingUpdate }}
         {{- toYaml . | nindent 6 }}
       {{- end }}
+  dnsPolicy: ClusterFirstWithHostNet
```

Comment:
shouldn't we set `hostNetwork: true` with the above option?

Reply:
I think the `hostNetwork` option is only to allow the pod to see the host network interface and being able to access it directly from the host, no?

Reply:
I mean, you're right in the sense that if you set `hostNetwork` to `true` then you have to set the `dnsPolicy` to `ClusterFirstWithHostNet` but I'm no sure about the reverse case?